doc/package-notes.xml

```diff
@@ -704,6 +704,42 @@ overrides = super: self: rec {
 }; }
 </programlisting>
   </para>
+  <para>
+    WeeChat allows to set defaults on startup using the <literal>--run-command</literal>.
+    The <literal>configure</literal> method can be used to pass commands to the program:
+<programlisting>weechat.override {
+  configure = { availablePlugins, ... }: {
+    plugins = builtins.attrValues availablePlugins;
+
+    init = ''
+      /set foo bar
+      /server add freenode chat.freenode.org
+      /even
+      /more
+      /commands
+    '';
+  };
+}</programlisting>
+    These values can be overridden manually when running
+    <literal>weechat --run-command "your-commands"</literal>.
```

Comment:
unfortunately not. I tried it out with multiple `--run-command` arguments. I can try to merge both values though, but I'm not sure, how easy this will be...

Reply:
It *is* a bit of complexity (especially for a bash script), but I think it is worth it because the alternative can be very confusing. This should do the trick (assuming `init` is set correctly):

```bash
#!/bin/sh
init="" # init script

new_args=()
# remove --run-command arguments and remember their commands
while [ $# -gt 0 ]; do
	if [[ "$1" == "--run-command" ]] || [[ "$1" == "-r" ]]; then
		# subsequent `--run-commands` override the previous ones, as is default weechat behaviour
		commands_to_run="$2"
		shift
	else
		new_args+=("$1")
	fi
	shift
done

new_args+=("--run-command" "$init;$commands_to_run")

exec weechat "${new_args[@]}"

```

Reply:
Oh I just saw that you already implemented that.